pylint/checkers/base.py

```diff
@@ -1525,15 +1525,26 @@ def _name_became_keyword_in_version(name, rules):
 
 class DocStringChecker(_BasicChecker):
     msgs = {
-        'C0111': ('Missing %s docstring', # W0131
-                  'missing-docstring',
-                  'Used when a module, function, class or method has no docstring.'
-                  'Some special methods like __init__ doesn\'t necessary require a '
-                  'docstring.'),
         'C0112': ('Empty %s docstring', # W0132
                   'empty-docstring',
                   'Used when a module, function, class or method has an empty '
                   'docstring (it would be too easy ;).'),
+        'C0114': ('Missing module docstring',
+                  'missing-module-docstring',
+                  'Used when a module has no docstring.'
+                  'Empty modules doesn\'t necessary require a docstring.',
```

Comment:
Can we fix the grammar here? Perhaps: "Empty modules do not require a docstring."